units/systemd-udevd.service.in

```diff
@@ -25,7 +25,7 @@ ExecStart=@rootlibexecdir@/systemd-udevd
 KillMode=mixed
 WatchdogSec=3min
 TasksMax=infinity
-MountFlags=slave
+PrivateMounts=yes
```

Comment:
I'm confused because this line-change seems to contradict `MountFlags` doc: `If not set – but file system namespaces are enabled through another file system namespace unit setting - <option>shared</option> mount propagation is used`. Is this a behavioral change, a doc bug or a misunderstanding of mine? Does this setting result in a single `MS_SLAVE` or in `MS_SLAVE`->`MS_SHARED`?

Reply:
previously we'd re-mount the mounts in the namespace to MS_SLAVE once. Now we remount to MS_SLAVE first and then to MS_SHARED. That way MS_SHARED is in effect afterwards, but the propagation is cut off already, as propagation once turned off can't be turned on again.

I'll extend the sentence you quoted a bit to point that out.